lib/common/index.d.ts

```diff
@@ -274,3 +272,25 @@ export let Mat33: {
   mulVec2(a: Mat33, b: Vec2): Vec2;
   add(a: Mat33, b: Mat33): Mat33;
 };
+
+export interface Sweep {
+  localCenter: Vec2;
+  c: Vec2;
+  a: number;
+  alpha0: number;
+  c0: Vec2;
+  a0: number;
+
+  setTransform(xf): void;
```

Comment:
I think you have to set the type of these parameters explicitly to `any`. Some people seemed to have problems otherwise when `noImplictAny` is set to true (but I have not tested it)

Reply:
yup, I forgot them.